docker/openproblems/Dockerfile

```diff
@@ -1,6 +1,19 @@
-FROM rackspacedot/python38:latest
+FROM python:3.8
```

Comment:
anything in particular different between the two images? This would  constistute quite a big change to the docker images.

Reply:
Talked it over with Scott, he didn't realize there was a base python image and he found the rackspacedot randomly just searching for python3.8 image

Reply:
okay, cool. Any difference in size?